### PR TITLE
Fix pagination to be more readable and also include last page

### PR DESCRIPTION
**Pull request by lucabelluccini**
Attempt to fix https://github.com/elastic/support-diagnostics/issues/599

### CODE REVIEW

File: src/main/java/co/elastic/support/diagnostics/commands/RunKibanaQueries.java [hunk 1]
```diff
@@ -133,40 +133,39 @@ private String getPageUrl(RestEntry action, int page, int perPage) {
...
-            // Get the first actions page
-            queries.add(getNewEntryPage(perPage, 1, action));
-            // If there is more pages add the new queries
-            if (perPage < total) {
-                int numberPages = (int)Math.ceil(total / perPage);
-                for (int i = 2; i <= numberPages; i++) {
-                    queries.add(getNewEntryPage(perPage, i, action));
-                }
+        String result = res.toString();
+        JsonNode root = JsonYamlUtils.createJsonNodeFromString(result);
```
Comment (pickypg) [inline src/main/java/co/elastic/support/diagnostics/commands/RunKibanaQueries.java:160]:
```suggestion
        JsonNode root = JsonYamlUtils.createJsonNodeFromString(res.toString());
```

**Review by pickypg — APPROVED** [review]
LGTM. Thanks for the tweak. I merged this with main and tweaked it a little myself.

**Comment by lucabelluccini** [comment]
Hello @pickypg 
The modifications are to ensure we have at least one file with 0 alerts right?
Thanks!

**Comment by pickypg** [comment]
> The modifications are to ensure we have at least one file with 0 alerts right?

Yep, that's right.

**Pull request merged**